backend-api/template.yaml

```diff
@@ -857,6 +857,27 @@ Resources:
               Service: lambda.amazonaws.com
             Action: sts:AssumeRole
       Policies:
+        - PolicyName: DynamoReadSessionsTable
+          PolicyDocument:
+            Version: "2012-10-17"
+            Statement:
+              - Effect: Allow
+                Action:
+                  - dynamodb:Query
+                Resource:
+                  - Fn::GetAtt: [ "SessionsTable", "Arn" ]
+                  - !Sub ${SessionsTable.Arn}/index/subjectIdentifier-timeToLive-index
+                Condition:
+                  ForAllValues:StringEquals:
```

Comment:
The list of attributes here is different from the list in the AsyncCredentialLambdaRole (?)

Reply:
Yes because this lambda also needs state and redirectUri?

Reply:
Correction: removing `state` as we need `sessionState` instead